.github/workflows/compressed-diff.yml

```
@@ -9,20 +9,31 @@ jobs:
       with:
         repository: 'bfintal/Stackable-Premium'
         ref: 'v3'
         path: 'pro__premium_only'
         token: '${{ secrets.ACCESS_KEY }}'
-    - name: Install Composer Dependencies
-      run: |
-        composer install --prefer-dist --no-progress --ignore-platform-reqs
-    - name: Setup Node # Fix because we can't do npm ci --legacy-peer-deps
-      uses: actions/setup-node@v1
+    - name: Setup Python
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)